test_scripts/API/PerformAudioPassThru/006_ATF_P_PerformAudioPassThru_WARNINGS_Different_Speech_Capabilities_SAPI_PHONEMES.lua

```diff
@@ -0,0 +1,73 @@
+---------------------------------------------------------------------------------------------
+-- Requirements summary:
+-- [PerformAudioPassThru] SDL must transfer request to HMI in case of valid "audioPassThruIcon" param
+-- [HMI API] UI.PerformAudioPassThru request/response
+-- [HMI API] TTS.Speak request/response
+-- [Mobile API] PerformAudioPassThru request/response
+-- [HMI_API] [MOBILE_API] The "audioPassThruIcon" param at "ImageFieldName" struct
+--
+-- Description:
+-- In case mobile app sends PerformAudioPassThru_request to SDL with:
+-- valid <audioPassThruIcon> parameter
+-- and the requested <audioPassThruIcon> exists at app`s sandbox (see AppStorageFolder section)
+-- as well as another related to request valid mandatory params
+-- SDL must transfer UI.PerformAudioPassThru (<audioPassThruIcon>, other params)_request + Speak_request (depends on parameters provided by the app) to HMI
+--1. Used preconditions
+--1.1 All parameters are present and within bounds speech capabilities is sent with type = "SAPI_PHONEMES"
+--1.2. AudioPassThruIcon exists at apps sub-directory of AppStorageFolder (value from ini file)
+--2. Performed steps
+-- Send PerformAudioPassThru (audioPassThruIcon, other params, ttsChunksType = "SAPI_PHONEMES") from mobile to SDL and check:
```

Comment:
@istoimenova Agree that this is part of another requirement <<[PerformAudioPassThru] requested "ttsChunks" is NOT supported>>, description is updated accordingly.
Please check in commit 596dba59820a3fd836183ee41cb912bd6519ba31